Core/HLE/sceNetAdhoc.cpp

```diff
@@ -3482,7 +3482,7 @@ void __NetTriggerCallbacks()
 
 			for (std::map<int, AdhocctlHandler>::iterator it = adhocctlHandlers.begin(); it != adhocctlHandlers.end(); ++it) {
 				args[2] = it->second.argument;
-				__KernelDirectMipsCall(it->second.entryPoint, NULL, args, 3, true);
+				__KernelDirectMipsCall(it->second.entryPoint, NULL, (u32*)args, 3, true);
```

Comment:
Hm.  Surely a pointer cast doesn't trigger swapping?

-[Unknown]

Reply:
that was just to silence the compiler heh. totally forgot about that.
args are probably never read again by native code, but for safety ill just change the type in all functions that come after this.